server/http.go

```diff
@@ -187,18 +188,21 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 
 func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 	if !p.verifyWebhookSecret(r) {
+		p.API.LogWarn("Could not verify webhook secreet")
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
 
 	if !strings.Contains(r.Header.Get("Content-Type"), "application/json") {
 		res := fmt.Sprintf("Expected Content-Type 'application/json' for webhook request, received '%s'.", r.Header.Get("Content-Type"))
+		p.API.LogWarn(res)
 		http.Error(w, res, http.StatusBadRequest)
 		return
 	}
 
 	b, err := ioutil.ReadAll(r.Body)
 	if err != nil {
+		p.API.LogWarn("Cannot read body from Webhook")
```

Comment:
Nit: `Warn` seams a bit to much, as this can happen regularly. The same comment applies for all other added log messages.

Reply:
Is it common to not being able to read the body of a Request? Isn't this a sign of something going wrong with the instance?

Reply:
My concern was more that an instance is misconfigured and the log messages would be to annoying. But maybe it's worth warning the admin a lot of about that.